__tests__/OmnichannelChatSDK.spec.ts

```diff
@@ -898,6 +898,30 @@ describe('Omnichannel Chat SDK', () => {
             expect(chatSDK.OCClient.sessionInit).toHaveBeenCalledTimes(0);
         });
 
+        it('ChatSDK.startChat() should not call OCClient.sessionInit() if OCClient.getChatToken() fails with 401 InvalidAuthentication', async () => {
```

Comment:
`ChatSDK.startChat() should fail with "InvalidAuthentication" if OCClient.getChatToken() fails with 401`

Reply:
edited description